apm-sniffer/apm-agent-core/src/main/java/org/skywalking/apm/agent/core/boot/DefaultNamedThreadFactory.java

```diff
@@ -32,8 +32,7 @@ public DefaultNamedThreadFactory(String name) {
     }
     @Override
     public Thread newThread(Runnable r) {
-        Thread t = new Thread();
-        t.setName(namePrefix + threadSeq.getAndIncrement());
+        Thread t = new Thread(null, r,namePrefix + threadSeq.getAndIncrement(),0);
```

Comment:
`Thread t = new Thread(r,namePrefix + threadSeq.getAndIncrement());`, this is enough?

Reply:
It is enough.It's come from DefaultThreadFactory which is the default ThreadFactory of Executors.